OnProfileUpdate/function.json

```diff
@@ -0,0 +1,23 @@
+{
+  "bindings": [
+    {
+      "type": "cosmosDBTrigger",
+      "name": "documents",
+      "direction": "in",
+      "connectionStringSetting": "COSMOSDB_CONNECTION_STRING",
+      "databaseName": "%COSMOSDB_NAME%",
+      "collectionName": "profiles",
+      "startFromBeginning": true,
+      "createLeaseCollectionIfNotExists": true,
+      "leaseCollectionName": "profile-emails-leases",
+      "leaseCollectionPrefix": "%ON_PROFILE_UPDATE_LEASES_PREFIX%-"
```

Comment:
This env variable should be added into the config to fail the health check if misconfigured or missing.